daft/expressions/expressions.py

```diff
@@ -657,8 +704,24 @@ def is_null(self) -> Expression:
         """Checks if values in the Expression are Null (a special value indicating missing data)
 
         Example:
-            >>> # [1., None, NaN] -> [False, True, False]
-            >>> col("x").is_null()
+            >>> import daft
+            >>> from numpy import nan
+            >>> df = daft.from_pydict({"x": [1., None, nan]})
```

Comment:
lets use `float("nan")` instead of numpy. For all the other ones as well thanks!